addons/web/static/src/js/views/graph/graph_view.js

```diff
@@ -75,6 +75,18 @@ var GraphView = AbstractView.extend({
             }
         });
 
+        const attrs = this.arch.attrs;
+        if (attrs.hasOwnProperty('groupBys')) {
```

Comment:
See comment on first commit, i think we are adding something that is already supported in another way, in graph and pivot (probably)